app/views/publish/courses/_basic_details_tab.html.erb

```diff
@@ -21,7 +21,7 @@
       <% unless course.further_education_course? %>
         <% summary_list.row(html_attributes: { data: { qa: "course__subjects" } }) do |row| %>
           <% row.key { "Subject".pluralize(course.subjects.count) } %>
-          <% row.value { course.sorted_subjects } %>
```

Comment:
`sorted_subjects`
Can that be dropped?

Reply:
No, we need it as we would revert back to it if the `master_subject_id` is `nil`

https://github.com/DFE-Digital/publish-teacher-training/pull/3113/files#diff-3803c296e70f15d1a8fafa24c3e9f46a5b22f3d88d3eaead20735ea3ddaf4a73R134-R135